### PR TITLE
feat/1.5/en/getting_started

### PR DESCRIPTION
**Pull request by yong313**
# Description (개요) 

Please include a summary of the change and which statement has been created or fixed.

## Keep in Mind

If you are creating a new statement, your first commit should always be a copy of the statement(model, ThanoSQL, function) template. This will allow reviewers to see what has been changed from the template, making the review process much more efficient.

## Type of change (작업사항)

Please delete options that are not relevant and check out the type of change.

- [ ] Model statmenet creation 
- [ ] ThanoSQL statmenet creation 
- [ ] Function statmenet creation 
- [ ] Model statement fix 
- [ ] ThanoSQL statement fix 
- [ ] Function statement fix 

Following templates can be found from the [Tech-wiki Docs](https://github.com/smartmind-team/tech-wiki/tree/main/project/thanosql/docs). 

## Writing Convention

Make sure to check out the [Writing Convention](https://github.com/smartmind-team/tech-wiki/blob/main/project/thanosql/docs/writing_convention.md). 

## Review Process

Review should be conducted by at least two people.

### CODE REVIEW

**Comment by ss-won** [comment]
ko ver: #206

File: docs/en/getting_started/paas/workspace/query_manager.md [hunk 1]
```diff
@@ -1,105 +1,202 @@
-# __Query Manager Manual__
+# **Query Manager Manual**
 
-The Query Manager provides a familiar UI similar to DBMS, allowing you to execute ThanoSQL queries and analyze the results. It provides an easy way to understand the schema and table structure of the workspace database, and facilitates the management of previous query results through Query Logs. Additionally, it offers a Data Viewer for analyzing unstructured data such as images and videos in the query execution results.
+The Query Manager enables you to execute ThanoSQL queries and analyze results through a user interface that is familiar and similar to a Database Management System (DBMS). It offers an easy way to understand the schema and table structure of the workspace database. You can effortlessly manage previous query results through query logs. Additionally, it provides a data viewer for analyzing unstructured data, such as images and videos, within query execution results.
```
Comment (stevensjcho) [inline docs/en/getting_started/paas/workspace/query_manager.md]:
```suggestion
The Query Manager enables you to execute ThanoSQL queries and analyze results through a user interface that is similar to a Database Management System (DBMS). It offers an intuitive way to understand the schema and table structure of the workspace database. You can effortlessly manage and view previous query results with query logs. Additionally, the Query Manager provides a data viewer for analyzing unstructured data, such as images and videos, within query execution results.

```

File: docs/en/getting_started/paas/workspace/query_manager.md [hunk 2]
```diff
@@ -1,105 +1,202 @@
...
 
-**Main Features**
+**Key Features**
 
-- Execution of ThanoSQL and basic SQL commands
-- Retrieval of data table schema information/preview
-- Query Log
-- Data Viewer
+- Execution of both ThanoSQL and basic SQL commands
```
Comment (stevensjcho) [inline docs/en/getting_started/paas/workspace/query_manager.md]:
```suggestion
- Execution of both ThanoSQL and regular SQL commands

```

File: docs/en/getting_started/paas/workspace/query_manager.md [hunk 3]
```diff
@@ -1,105 +1,202 @@
...
+[![IMAGE](../../../img/getting_started/paas/workspace/qm_img_2.png){: style="max-height:none"}](../../../img/getting_started/paas/workspace/qm_img_2.png)
 
-① Tab : You can write queries and view results in separate tabs.
+① Tabs: Write queries and view results in different tabs. You can add or delete tabs as needed.
 
-② Query Editor : You can write and execute ThanoSQL or basic SQL queries. It supports [Keyboard Shortcuts](#keyboard-shortcuts).
+② Query Input Area: Input queries and execute/stop them. Supports [Keyboard Shortcuts](#keyboard-shortcuts).
 
-③ CSV Download : You can download the results of the query as a CSV file.
+③ CSV Download: Download result tables in CSV format.
```
Comment (stevensjcho) [inline docs/en/getting_started/paas/workspace/query_manager.md]:
```suggestion
③ CSV Download: Download result as a CSV file.

```

File: docs/en/getting_started/paas/workspace/query_manager.md [hunk 4]
```diff
@@ -1,105 +1,202 @@
...
 
-④ [Data Viewer](#data-viewer) : This feature allows you to analyze unstructured data, such as images and videos.
+④ [Data Viewer](#data-viewer) : Used for analyzing unstructured data like images and videos.
 
-⑤ Result Table : The query results are displayed in a table format.
+⑤ Result Table: Displays query results in a table format.
```
Comment (stevensjcho) [inline docs/en/getting_started/paas/workspace/query_manager.md]:
```suggestion
⑤ Result Table: Displays query results as a table.

```

File: docs/en/getting_started/paas/workspace/query_manager.md [hunk 5]
```diff
@@ -1,105 +1,202 @@
...
-| Action | Key Combination (Windows/Linux) | Key Combination (Mac OS) |
-| -- | -- | -- |
-| Run current query or selected query | Ctrl+Enter | Cmd + Enter |
-| Autocomplete current word | Tab or Ctrl + Space | Tab or Cmd + Space |
-| Comment current or selected lines | Ctrl+/ | Cmd + / |
-| Open the menu of the editor | Shift + F10 | Shift + F10 |
-| Open the list of shortcuts for the editor | F1 | F1 |
+| Action                                        | Key Combination (Windows/Linux) | Key Combination (Mac OS) |
+| --------------------------------------------- | ------------------------------- | ------------------------ |
+| Execute the current or selected query         | Ctrl+Enter                      | Cmd + Enter              |
```
Comment (stevensjcho) [inline docs/en/getting_started/paas/workspace/query_manager.md]:
```suggestion
| Execute the current or highlighted query         | Ctrl+Enter                      | Cmd + Enter              |

```

File: docs/en/getting_started/paas/workspace/query_manager.md [hunk 6]
```diff
@@ -1,105 +1,202 @@
...
+| Auto-complete the current word                | Tab or Ctrl + Space             | Tab or Cmd + Space       |
+| Add a comment to the current or selected line | Ctrl+/                          | Cmd + /                  |
```
Comment (stevensjcho) [inline docs/en/getting_started/paas/workspace/query_manager.md:55]:
```suggestion
| Add a comment to the current or highlighted line | Ctrl+/                          | Cmd + /                  |

```

File: docs/en/getting_started/paas/workspace/query_manager.md [hunk 7]
```diff
@@ -1,105 +1,202 @@
...
 
-You can check the workspace DB [Schema](https://www.postgresql.org/docs/current/ddl-schemas.html) and table configuration information. Each workspace basically provides `public` and `qm` schemas, and all queries that do not specify schema refer to `public` tables. Among the queries executed in the query editor, all queries whose results are in the form of a table are stored in the `qm` schema as an arbitrary table name. Using this, previous table execution results can be reused at any time.
+### **Database Explorer**
 
-!!! warning
-    Query result temporary tables in schema `qm` are only kept for a maximum of 7 days.
+View schema and table configuration information of the workspace database. Each workspace provides default 'public' and temporary result tables. Queries that do not specify a schema refer to tables in the 'public' schema. Any queries with results in tabular form are saved in temporary result tables with arbitrary names. This allows you to reuse previous table execution results at any time.
```
Comment (stevensjcho) [inline docs/en/getting_started/paas/workspace/query_manager.md]:
```suggestion
The Database Explorer shows the schema and table configuration information of the workspace database. Each workspace provides a default 'public' schema and temporary result tables. Queries that do not specify a schema refer to tables in the 'public' schema by default. Additionally, table results are saved as temporary result tables with arbitrary names. This allows you to reuse and view previous table execution results.

```

File: docs/en/getting_started/paas/workspace/query_manager.md [hunk 8]
```diff
@@ -1,105 +1,202 @@
...
-You can check the column names, data types, and whether null values are allowed for the column.
+View column names, data types, and nullability of columns in the data table.
 
-[![IMAGE](../../../img/getting_started/paas/workspace/img3.png){: style="max-height:none"}](../../../img/getting_started/paas/workspace/img3.png)
+[![IMAGE](../../../img/getting_started/paas/workspace/qm_img_14.png){: style="max-height:none"}](../../../img/getting_started/paas/workspace/qm_img_14.png)
 
 **Preview**
 
-You can quickly understand the contents of the data table in question.
+Quickly grasp the content of the data table.
```
Comment (stevensjcho) [inline docs/en/getting_started/paas/workspace/query_manager.md]:
```suggestion
Quickly understand the content of the data table.

```

grasp라는 단어 어색함

File: docs/en/getting_started/paas/workspace/query_manager.md [hunk 9]
```diff
@@ -1,105 +1,202 @@
...
-### __Query Log__
+#### **Adding Data**
 
-All queries executed in Query Manager are logged and displayed in the query log section. They are listed in the order of the query creation time, showing the query statement and its execution status.
+You can add new tables and schemas to the database.
 
-[![IMAGE](../../../img/getting_started/paas/workspace/qm_img_2.png){: style="max-height:none"}](../../../img/getting_started/paas/workspace/qm_img_2.png)
+##### **Adding Tables**
+
+Click the **`+`** button located to the right of the Database in the database explorer to open the menu for adding table data, then click 'Add Table'.
```
Comment (stevensjcho) [inline docs/en/getting_started/paas/workspace/query_manager.md]:
```suggestion
Click the **`+`** button located to the right of the Database in the database explorer to open the add table menu, then click 'Add Table'.

```

File: docs/en/getting_started/paas/workspace/query_manager.md [hunk 10]
```diff
@@ -1,105 +1,202 @@
...
+
+[![IMAGE](../../../img/getting_started/paas/workspace/qm_img_3.png){: style="max-height:none"}](../../../img/getting_started/paas/workspace/qm_img_3.png)
+
+**Create a New Table Directly**
+
+After opening the add table window, select 'Create a New Table Directly' and click Next.
```
Comment (stevensjcho) [inline docs/en/getting_started/paas/workspace/query_manager.md]: 여기서 window는 위에 menu를 말하는 건가요?? 그렇다면 window -> menu로 바꿔주세요

File: docs/en/getting_started/paas/workspace/query_manager.md [hunk 11]
```diff
@@ -1,105 +1,202 @@
...
+
+[![IMAGE](../../../img/getting_started/paas/workspace/qm_img_5.png){: style="max-height:none"}](../../../img/getting_started/paas/workspace/qm_img_5.png)
+
+[![IMAGE](../../../img/getting_started/paas/workspace/qm_img_6.png){: style="max-height:none"}](../../../img/getting_started/paas/workspace/qm_img_6.png)
+
+① Select Schema: Choose the schema to create the table in.
+
+② Create Schema: Create a new schema.
+
+③ Name(table): Enter the name for the table to be created. Table names can include letters, numbers, and underscores, but cannot start with a number. Duplicate table names cannot be created within the same schema.
```
Comment (stevensjcho) [inline docs/en/getting_started/paas/workspace/query_manager.md]:
```suggestion
③ Name(table): Enter the name of the table to be created. Table names can include letters, numbers, and underscores, but cannot start with a number. Duplicate table names cannot be created within the same schema.

```

File: docs/en/getting_started/paas/workspace/query_manager.md [hunk 12]
```diff
@@ -1,105 +1,202 @@
...
+
+④ Name(column): Enter the column name. This is a required field, and duplicate column names cannot be created within the same table.
```
Comment (stevensjcho) [inline docs/en/getting_started/paas/workspace/query_manager.md]:
```suggestion
④ Name(column): Enter the column name. Duplicate column names cannot be created within the same table. This is a required field.

```

File: docs/en/getting_started/paas/workspace/query_manager.md [hunk 13]
```diff
@@ -1,105 +1,202 @@
...
+
+⑤ Data Type: Select the data type for the column. This is a required field.
+
+⑥ Column Options: Depending on the Data Type, options can be adjusted, and settings can be customized according to the column's purpose.
```
Comment (stevensjcho) [inline docs/en/getting_started/paas/workspace/query_manager.md]:
```suggestion
⑥ Column Options: Depending on the Data Type, the options can be adjusted and customized accordingly.

```

File: docs/en/getting_started/paas/workspace/query_manager.md [hunk 14]
```diff
@@ -1,105 +1,202 @@
...
+
+⑦ Column Delete Button: Delete the column.
+
+⑧ Column Add Button: Add a new column.
+
+⑨ Cancel Button: Cancel creating a new table directly.
```
Comment (stevensjcho) [inline docs/en/getting_started/paas/workspace/query_manager.md]:
```suggestion
⑨ Cancel Button: Cancel creating a new table.

```

File: docs/en/getting_started/paas/workspace/query_manager.md [hunk 15]
```diff
@@ -1,105 +1,202 @@
...
+
+⑩ Complete Button: Complete creating a new table directly.
```
Comment (stevensjcho) [inline docs/en/getting_started/paas/workspace/query_manager.md]:
```suggestion
⑩ Complete Button: Complete directly creating a new table.

```

File: docs/en/getting_started/paas/workspace/query_manager.md [hunk 16]
```diff
@@ -1,105 +1,202 @@
...
+
+① File Format Selection: Choose between CSV and Excel. For CSV, only .csv files can be uploaded. For Excel, only .xls, .xlsx, .xlsm, .xlsb, .odf, .ods, and .odt files can be uploaded.
+
+② File Upload: Upload the file according to the selected format.
+
+③ Schema Selection: Choose the schema in which to create the table.
+
+④ Create Schema: Press the Create Schema button to create a new schema, useful for initial usage or when you want to create a new schema.
+
+⑤ Name: Enter the name for the table to be created. Table names can include letters, numbers, and underscores, but cannot start with a number. Duplicate table names cannot be created within the same schema.
```
Comment (stevensjcho) [inline docs/en/getting_started/paas/workspace/query_manager.md]:
```suggestion
⑤ Name: Enter the name of the table to be created. Table names can include letters, numbers, and underscores, but cannot start with a number. Duplicate table names cannot be created within the same schema.

```

File: docs/en/getting_started/paas/workspace/query_manager.md [hunk 17]
```diff
@@ -1,105 +1,202 @@
...
+
+⑥ Column Settings
+
+You can automatically detect the columns in the uploaded file or manually input them.
```
Comment (stevensjcho) [inline docs/en/getting_started/paas/workspace/query_manager.md]:
```suggestion
Columns can be automatically detected in the uploaded file or you can manually input them.

```

File: docs/en/getting_started/paas/workspace/query_manager.md [hunk 18]
```diff
@@ -1,105 +1,202 @@
...
+
+- Data Writing: Select data writing options.
+
+⑧ Cancel Button: Cancel local file upload.
+
+⑨ Complete Button: Complete local file upload.
+
+##### **Create Schemas**
+
+You can create a new schema.
```
Comment (stevensjcho) [inline docs/en/getting_started/paas/workspace/query_manager.md]:
```suggestion
Create a new schema.

```

File: docs/en/getting_started/paas/workspace/query_manager.md [hunk 19]
```diff
@@ -1,105 +1,202 @@
...
+
+Click the **`+`** button located to the right of the Database in the database explorer to open the menu for adding table data, then click 'Create Schema'.
```
Comment (stevensjcho) [inline docs/en/getting_started/paas/workspace/query_manager.md]:
```suggestion
Click the **`+`** button located to the right of the Database in the database explorer to open the add table menu, then click 'Create Schema'.

```

File: docs/en/getting_started/paas/workspace/query_manager.md [hunk 20]
```diff
@@ -1,105 +1,202 @@
...
+
+[![IMAGE](../../../img/getting_started/paas/workspace/qm_img_11.png){: style="max-height:none"}](../../../img/getting_started/paas/workspace/qm_img_9.png)
+
+[![IMAGE](../../../img/getting_started/paas/workspace/qm_img_12.png){: style="max-height:none"}](../../../img/getting_started/paas/workspace/qm_img_12.png)
+
+① Schema Name: Enter the name for the schema to be created. Schema names can include letters, numbers, and underscores, but cannot start with a number. Duplicate schema names cannot be created within the same database.
```
Comment (stevensjcho) [inline docs/en/getting_started/paas/workspace/query_manager.md]:
```suggestion
① Schema Name: Enter the name of the schema to be created. Schema names can include letters, numbers, and underscores, but cannot start with a number. Duplicate schema names cannot be created within the same database.

```

File: docs/en/getting_started/paas/workspace/query_manager.md [hunk 21]
```diff
@@ -1,105 +1,202 @@
...
+
+③ Complete Button: Complete schema creation.
+
+#### Delete Table
+
+[![IMAGE](../../../img/getting_started/paas/workspace/qm_img_13.png){: style="max-height:none"}](../../../img/getting_started/paas/workspace/qm_img_13.png)
+
+### **Query Logs**
+
+All queries executed in the Query Manager are logged and can be viewed in the query logs section. They are listed in descending order of creation time. The logs show the executed query statement and whether the query was successful.
```
Comment (stevensjcho) [inline docs/en/getting_started/paas/workspace/query_manager.md]:
```suggestion
All queries executed in the Query Manager are logged and can be viewed in the query logs section. They are listed in descending order of creation time. The logs show the executed query statement and whether the execution was successful.

```

File: docs/en/getting_started/paas/workspace/query_manager.md [hunk 22]
```diff
@@ -1,105 +1,202 @@
...
+
+[![IMAGE](../../../img/getting_started/paas/workspace/qm_img_16.png){: style="max-height:none"}](../../../img/getting_started/paas/workspace/qm_img_16.png)
+
+#### Query Search
+
+When you input a query statement, it searches for log entries with similar query statements.
```
Comment (stevensjcho) [inline docs/en/getting_started/paas/workspace/query_manager.md]:
```suggestion
When you input a query statement, it searches for log entries that contain the given query statement.

```

**Pull request merged**